StardewArchipelago/Stardew/StardewItemManager.cs

```diff
@@ -178,6 +179,16 @@ public StardewObject GetObjectByName(string itemName)
             throw new ArgumentException($"Item not found: {itemName}");
         }
 
+        public List<StardewObject> GetObjectsByColor(string color)
+        {
+            if (_objectsByColor.ContainsKey(color))

```

Comment:
the _objectsByColor are never initialized.
I understand you didn't write the code to fill the dictionary yet, but you should at least initialize it, even if empty, so that the method at least doesn't crash instantly